gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -592,6 +593,41 @@ ChannelCredentials createS2ASecuredChannelCredentials() {
     }
   }
 
+  boolean isMtlsS2AHardBoundTokensEnabled() {
+    if (!useS2A) {
+      // If S2A cannot be used, {@code HardBoundTokenTypes.MTLS_S2A} hard bound tokens should not be
+      // used
+      return false;
+    }
+    if (allowedHardBoundTokenTypes == null
```

Comment:
nit: I think this may have slipped in a previous review.

Is it possible to have allowedHardBoundTokenTypes default to a empty list? And we ensure that the setter can't set this value to be null? I think I would rather have something like an list.isEmpty() check rather than a null check

Reply:
Yes, we can have `allowedHardBoundTokenTypes` default to an empty list and make sure if someone passes null to setter, we set it to an empty list instead. Done in https://github.com/googleapis/sdk-platform-java/pull/3591/commits/7c20422a57882b0243b298ef8e6f33293e4aa61a